File: Habitica/src/main/java/com/habitrpg/android/habitica/models/user/SubscriptionPlanConsecutive.java
package com.habitrpg.android.habitica.models.user;

import io.realm.RealmObject;

public class SubscriptionPlanConsecutive extends RealmObject {

    SubscriptionPlan subscriptionPlan;
    private int trinkets;
    private int gemCapExtra;
    private int offset;
    private int count;

    public int getTrinkets() {
        return trinkets;
    }

    public void setTrinkets(int trinkets) {
        this.trinkets = trinkets;
    }

    public int getGemCapExtra() {
        return gemCapExtra;
    }

    public void setGemCapExtra(int gemCapExtra) {
        this.gemCapExtra = gemCapExtra;
    }

    public int getOffset() {
        return offset;
    }

    public void setOffset(int offset) {
        this.offset = offset;
    }

    public int getCount() {
        return count;
    }

    public void setCount(int count) {
        this.count = count;
    }
}

File: Habitica/src/main/java/com/habitrpg/android/habitica/models/user/SubscriptionPlanConsecutive.kt
package com.habitrpg.android.habitica.models.user

import com.habitrpg.android.habitica.models.BaseObject
import io.realm.RealmObject
import io.realm.annotations.RealmClass

@RealmClass(embedded = true)
open class SubscriptionPlanConsecutive : RealmObject(), BaseObject {
    var trinkets = 0
    var gemCapExtra = 0
    var offset = 0
    var count = 0
}
